NET01.1Solution/NET01.1Task/Entity.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NET01._1Task
```

Comment:
Follow PascalCase for namespace naming https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-namespaces.